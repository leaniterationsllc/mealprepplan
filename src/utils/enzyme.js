/**
 * Copyright (c) 2017-present, Lean Iterations LLC.
 * All rights reserved.
 *
 * This source code is licensed under the MIT-style license found in the
 * LICENSE file in the root directory of this source tree.
 */
// configuring enzyme to use the adapter for React 16
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

export default Enzyme.configure({ adapter: new Adapter() });
