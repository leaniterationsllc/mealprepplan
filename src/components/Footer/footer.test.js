/**
 * Copyright (c) 2017-present, Lean Iterations LLC.
 * All rights reserved.
 *
 * This source code is licensed under the MIT-style license found in the
 * LICENSE file in the root directory of this source tree.
 */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from './index';
import enzymeconf from '../../utils/enzyme';

describe('Footer', () => {
  it('renders correctly', () => {
    const tree = shallow(<Footer />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
