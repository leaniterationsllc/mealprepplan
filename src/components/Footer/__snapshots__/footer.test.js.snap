// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div
  className="css-16xh8je"
>
  <Styled(div)
    align="left"
    color="#226666"
    p={
      Array [
        3,
        3,
        4,
      ]
    }
  >
    <Styled(div)
      justify="space-between"
      wrap={
        Array [
          "wrap",
          "wrap",
          "nowrap",
        ]
      }
    >
      <Styled(div)
        width={
          Array [
            1,
            1,
            0.3333333333333333,
          ]
        }
      >
        <Styled(div)>
          <div>
            <p>
              Meal Prep Planner and Diary
            </p>
          </div>
        </Styled(div)>
      </Styled(div)>
      <Styled(div)
        width={
          Array [
            1,
            1,
            0.3333333333333333,
          ]
        }
      >
        <Styled(div)>
          <ul
            className="css-1epioiy"
          >
            <li>
              <GatsbyLink
                to="/"
              >
                Home
              </GatsbyLink>
            </li>
            <li>
              <GatsbyLink
                to="/store"
              >
                Store
              </GatsbyLink>
            </li>
            <li>
              <GatsbyLink
                to="/blog"
              >
                Blog
              </GatsbyLink>
            </li>
            <li>
              <GatsbyLink
                to="/privacy"
              >
                Privacy Policy
              </GatsbyLink>
            </li>
          </ul>
        </Styled(div)>
      </Styled(div)>
      <Styled(div)
        width={
          Array [
            1,
            1,
            0.3333333333333333,
          ]
        }
      >
        <Styled(div)>
          <ul
            className="css-1jsczpv"
          >
            <li>
              <a
                href="https://www.facebook.com/groups/550109412047209/"
                rel="noopener"
                target="_blank"
                title="Link to our Facebook page"
              >
                <div
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "<svg xmlns=\\"http://www.w3.org/2000/svg\\" width=\\"30\\" height=\\"30\\" viewBox=\\"0 0 24 24\\" fill=\\"none\\" stroke=\\"currentColor\\" stroke-width=\\"2\\" stroke-linecap=\\"round\\" stroke-linejoin=\\"round\\" class=\\"css-xvpfj7 feather feather-facebook\\"><path d=\\"M18 2h-3a5 5 0 0 0-5 5v3H7v4h3v8h4v-8h3l1-4h-4V7a1 1 0 0 1 1-1h3z\\"></path></svg>",
                    }
                  }
                />
              </a>
            </li>
            <li>
              <a
                href="https://www.instagram.com/mealprepapp/"
                rel="noopener"
                target="_blank"
                title="Link to out Instragram account"
              >
                <div
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "<svg xmlns=\\"http://www.w3.org/2000/svg\\" width=\\"30\\" height=\\"30\\" viewBox=\\"0 0 24 24\\" fill=\\"none\\" stroke=\\"currentColor\\" stroke-width=\\"2\\" stroke-linecap=\\"round\\" stroke-linejoin=\\"round\\" class=\\"css-xvpfj7 feather feather-instagram\\"><rect x=\\"2\\" y=\\"2\\" width=\\"20\\" height=\\"20\\" rx=\\"5\\" ry=\\"5\\"></rect><path d=\\"M16 11.37A4 4 0 1 1 12.63 8 4 4 0 0 1 16 11.37z\\"></path><line x1=\\"17.5\\" y1=\\"6.5\\" x2=\\"17.5\\" y2=\\"6.5\\"></line></svg>",
                    }
                  }
                />
              </a>
            </li>
            <li>
              <a
                href="mailto:support@leaniterationsllc.com"
                title="Our E-Mail address"
              >
                <div
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "<svg xmlns=\\"http://www.w3.org/2000/svg\\" width=\\"30\\" height=\\"30\\" viewBox=\\"0 0 24 24\\" fill=\\"none\\" stroke=\\"currentColor\\" stroke-width=\\"2\\" stroke-linecap=\\"round\\" stroke-linejoin=\\"round\\" class=\\"css-xvpfj7 feather feather-mail\\"><path d=\\"M4 4h16c1.1 0 2 .9 2 2v12c0 1.1-.9 2-2 2H4c-1.1 0-2-.9-2-2V6c0-1.1.9-2 2-2z\\"></path><polyline points=\\"22,6 12,13 2,6\\"></polyline></svg>",
                    }
                  }
                />
              </a>
            </li>
          </ul>
        </Styled(div)>
      </Styled(div)>
    </Styled(div)>
  </Styled(div)>
  <Styled(div)
    align="center"
    px={
      Array [
        3,
        3,
        1,
      ]
    }
  >
    <Styled(p)>
      Copyright Â© 2017 Lean Iterations LLC. All rights reserved.
    </Styled(p)>
  </Styled(div)>
</div>
`;
